podcasts/End of Year/Stories/Views/PodcastCover.swift

```diff
@@ -44,7 +41,7 @@ struct PodcastCover: View {
                 }
             }
             .opacity(image != nil ? 1 : 0.2)
-            .blendMode(!renderForSharing && image != nil ? .multiply : .normal)
+            .blendMode(.multiply)
```

Comment:
I had to change that because with the opacity applied to the artworks, the generated image had a strange "white" (like if the background below them was white):

<img src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/e0924d61-80d1-4a3c-89c9-7bea436442d3" width="300">

It seems we did this change to prevent an issue with shadows that, as far as I can see, doesn't happen anymore.